.template/addons/crud/app/assets/stylesheets/layouts/_application.scss

```
@@ -0,0 +1,92 @@
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.app {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    justify-content: space-between;
+  }
+
+  &__main {
+    padding: rem(16px);
+    margin: rem(16px);
+  }
+
+  &__footer {
+    grid-area: footer;
+    padding: rem(8px);
+  }
+
+  &__nav_bar_top {
```

I think we should separate them by the hyphen (-) instead
```suggestion
  &__nav-bar-top {
```

---

3270c1d :white_check_mark: